agent/tcs/handler/types.go

```diff
@@ -50,3 +51,10 @@ func (params *TelemetrySessionParams) time() ttime.Time {
 	})
 	return params._time
 }
+
+func (params *TelemetrySessionParams) isMetricsDisabled() (bool, error) {
```

Comment:
The function name makes me a little bit confusing. It is called isMetricsDisabled, but when DisableMetrics flag is true, the function will not return true. Do you want to say MetricsSessionDisabled or something? Maybe change the function name and add some description as comments here.

Reply:
fair. changed to `isContainerHealthMetricsDisabled` since checking against both `DisabledMetrics && DisableDockerHealthCheck`